plugins/vpp/ifplugin/vppcalls/vpp1908/dump_interface_vppcalls.go

```diff
@@ -161,6 +148,130 @@ func (h *InterfaceVppHandler) dumpInterfaces(ifIdxs ...uint32) (map[uint32]*vppc
 	return ifs, nil
 }
 
+func (h *InterfaceVppHandler) DumpInterfaces2(ifIdxs ...uint32) (map[uint32]*vppcalls.InterfaceDetails, error) {
```

Comment:
Did you forget to finish this one? The implementation looks the same as  `DumpInterfaces` except the index check and I did not find where is it called. If both of them were intended to remain here, please remove duplicated code and find some more suitable name, this does not look good.